src/docs/components/InstallationDocs/DocCollapsible.tsx

```diff
@@ -0,0 +1,16 @@
+import {
+  FadingCollapsible,
+  FadingCollapsibleProps,
+} from "@components/FadingCollapsible";
+
+const defaultOptions: Omit<FadingCollapsibleProps, "children"> = {
+  expandButtonEl: "Show more",
+  collapseButtonEl: "Hide section",
+  className: "space-y-8 md:space-y-12 border-l border-stone-700 pl-2 md:pl-4",
```

Comment:
This border left looks a bit off. I'd remove it and just leave the background gradient.